### PR TITLE
Add unused CSS reporting

### PR DESCRIPTION
**Pull request by fverloop**
I've used PurifyCSS to add CSS checking to the report task

### CODE REVIEW

**Comment by fverloop** [comment]
Only way I get the task to run with `run.js` though,  is by compiling `gulpfile.coffee` manually.

**Comment by koenbok** [comment]
🔥🔥🔥 let me check.

File: gulpfile.coffee [hunk 1]
```diff
@@ -282,12 +283,18 @@ gulp.task "server", (cb) ->
 			cb(err)
 
 gulp.task "report", ->
-	for ext in ["html", "css", "jpg", "png", "mp4", "ico"]
```
Comment (koenbok) [inline gulpfile.coffee]: No love for ico?

File: gulpfile.coffee [hunk 2]
```diff
@@ -282,12 +283,18 @@ gulp.task "server", (cb) ->
...
+	for ext in ["html", "css", "jpg", "png", "mp4"]
 		gutil.log(gutil.colors.green("#{ext} #{getTotalSizeForFileType(buildPath(paths.assets), ext)}"))
 
+	gutil.log(gutil.colors.green("Checking unused CSS selectors..."))
```
Comment (koenbok) [inline gulpfile.coffee]: This seems like the right place.

**Comment by koenbok** [comment]
Add the package to npm: `Error: Cannot find module 'gulp-purifycss'`.

**Comment by fverloop** [comment]
Shoot — forgot `--save` when installing package.

File: gulpfile.coffee (hunk 1 above)
Comment (fverloop) [inline gulpfile.coffee]: We're not really using any ico files. Is this really important?

**Comment by koenbok** [comment]
Watch the whitespace: 
image

**Comment by koenbok** [comment]
I really don't like how the output looks like, but it's hard to hack around at this point. So. Congrats!

**Comment by fverloop** [comment]
🎉

**Pull request merged**